### ISSUE TITLE
create a mock /app/api/admin/status API endpoint

### ISSUE THREAD

**Issue by vakopian** [cornsuite/imagepop-ui#14]
When a GET request comes in to the back-end server (e.g. the server running on port 3001 from #13 ) with the `/app/api/admin/status` endpoint the server reply should be something like this
```json
[
  {
    "name": "apiVersion",
    "value": "1.0.0"
  },
  {
    "name": "uptime",
    "value": "13 hours"
  },
  {
    "name": "processId",
    "value": "4612"
  },
  {
    "name": "serverHostName",
    "value": "localhost"
  },
]
```

**Comment by vakopian** [comment]
Please note, this API endpoint is (for now) just a way to make sure the back-end server works and also servers as a simple example on how mock API endpoints can be implemented. The actual values returned from the back-end can be hard-coded (since the back-end is mocked), so no need to spend time in figuring out how the get the process id or host name or any such value.
It might also help to install the PostMan chrom extension (https://chrome.google.com/webstore/detail/postman/fhbjgbiflinjbdggehcddcbncdddomop?hl=en) to be able to test this and other API endpoints later on

**Comment by vakopian** [comment]
fixed in #17
